readable.js

```diff
@@ -1,9 +1,10 @@
-(function (){
+var Stream = (function (){
   try {
-    exports.Stream = require('st' + 'ream'); // hack to fix a circular dependency issue when used with browserify
+    return require('st' + 'ream'); // hack to fix a circular dependency issue when used with browserify
   } catch(_){}
 }());
 exports = module.exports = require('./lib/_stream_readable.js');
+exports.Stream = Stream || exports;
```

Comment:
`exports.Stream = exports`

So, you could have `exports.Stream.Stream.Stream.Stream === exports`, wait what?